docs/dev/communication.md

```diff
@@ -0,0 +1,15 @@
+# Communication Guidelines
+
+pyDARN is under the Data Analysis Working Group that has setup some guidelines on [communication](https://superdarn.github.io/dawg/documents/communication-guidelines/) to encourage positive collaborative environment.
+Please follow these guidelines when commenting on GitHub. 
+
+Also please read workflow documentation on issues and [pull requests](dev/PR.md) to communicate properly for collaborative work.
+
+## Disputes Procedure
+
+If one of the members have a dispute that violates the communication guidelines and is creating a negative environment in the pyDARN community, please follow the procedure for resolving the conflict:
+1. Please try resolve the matter between each member offline.
+2. If the dispute cannot be resolved or environment has become toxic and overwhelming please contact the [lead developer for pyDARN](marina.t.schmidt@gmail.com).
+3. If the dispute cannot be resolved, involves the lead developer, or breaches the [DAWG charter](https://superdarn.github.io/dawg/reports/202009/charter-update) please contact the chairs of DAWG.
```

Comment:
Do we want to add E and K's emails here?

Reply:
@ksterne and  @ecbland ?

I was not sure so I left it blank